### ISSUE TITLE
Tree: Keyboard navigation breaks due to missing $this while pressing down arrow

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#8336]
PF 11

<img width="574" alt="Screenshot 2022-01-26 at 17 08 53" src="https://user-images.githubusercontent.com/7500178/151201168-eaf1a038-4f0b-4b3d-bde9-03c81e53076b.png">

Can also be seen at

https://www.primefaces.org/showcase/ui/data/tree/selection.xhtml

**Comment by melloware** [comment]
Nice looks like an easy fix :)

**Comment by jepsar** [comment]
@melloware Yeah, easiest fix I did so far
